### ISSUE TITLE
训练损失从1.4下降到了0.5，训练5个epoch了，从开始到现在验证集ACCURACY一直是64%

### ISSUE THREAD

**Issue by kunzeng-ch** [LlamaFamily/Llama2-Chinese#289]
使用lora训练llama-2-7b-chat，数据是博主data目录下的数据，代码并无改动，启动的脚本用的是finetune_lora.sh，deepspeed启动命令如下：
deepspeed --include localhost:0 finetune_clm_lora.py \
    --model_name_or_path /home/zengshaokun/models/llama/llama-2-7b-chat \
    --train_files ../../data/train_sft.csv \
    --validation_files  ../../data/dev_sft.csv \
                         ../../data/dev_sft_sharegpt.csv \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 4 \
    --do_train \
    --do_eval \
    --use_fast_tokenizer false \
    --output_dir ${output_model} \
    --evaluation_strategy  steps \
    --max_eval_samples 800 \
    --learning_rate 1e-4 \
    --gradient_accumulation_steps 4 \
    --num_train_epochs 10 \
    --warmup_steps 400 \
    --load_in_bits 8 \
    --lora_r 8 \
    --lora_alpha 32 \
    --target_modules q_proj,k_proj,v_proj,o_proj,down_proj,gate_proj,up_proj \
    --logging_dir ${output_model}/logs \
    --logging_strategy steps \
    --logging_steps 10 \
    --save_strategy steps \
    --preprocessing_num_workers 10 \
    --save_steps 20 \
    --eval_steps 20 \
    --save_total_limit 20 \
    --seed 42 \
    --disable_tqdm false \
    --ddp_find_unused_parameters false \
    --block_size 2048 \
    --report_to tensorboard \
    --overwrite_output_dir \
    --deepspeed ds_config_zero2.json \
    --ignore_data_skip true \
    --gradient_checkpointing \
    --ddp_timeout 18000000 \
    | tee -a ${output_model}/train.log

**Comment by ShuFengMuYu** [comment]
你这种可能是出现了过拟合吧
